rules/CodingStyle/ClassNameImport/ClassNameImportSkipVoter/UsesClassNameImportSkipVoter.php

```diff
@@ -26,7 +26,7 @@ public function __construct(
 
     public function shouldSkip(File $file, FullyQualifiedObjectType $fullyQualifiedObjectType, Node $node): bool
     {
-        $useImportTypes = $this->useNodesToAddCollector->getUseImportTypesByNode($file);
+        $useImportTypes = $this->useNodesToAddCollector->getUseImportTypes();
```

Comment:
It's not clear what the method returns now.  Probably depends on some inner order, which is not good idea.